src/main/java/com/minecolonies/coremod/blocks/BlockConstructionTapeCorner.java

```diff
@@ -0,0 +1,339 @@
+     * End of the collision box facing South.
+     */
+    private static final double S_END_COLLISION_X = 1.0;
+
+    /**
+     * Start of the collision box at z facing South.
+     */
+    private static final double S_START_COLLISION_Z = 0.4375;
+
+    /**
+     * End of the collision box facing South.
+     */
+    private static final double S_END_COLLISION_Z = 1.0;
+
+    /**
+     * Start of the collision box at x facing East.
+     */
+    private static final double E_START_COLLISION_X = 0.4375;
+
+    /**
+     * End of the collision box facing East.
+     */
+    private static final double E_END_COLLISION_X = 1.0;
+
+    /**
+     * Start of the collision box at z facing East.
+     */
+    private static final double E_START_COLLISION_Z = 0.0;
+
+    /**
+     * End of the collision box facing East.
+     */
+    private static final double E_END_COLLISION_Z = 0.5625;
+
+    /**
+     * Height of the collision box.
+     */
+    private static final double HEIGHT_COLLISION = 1.0;
+
+    /**
+     * How much light goes through the block.
+     */
+    private static final int LIGHT_OPACITY = 0;
+
+    /**
+     * Constructor for the Substitution block.
+     * sets the creative tab, as well as the resistance and the hardness.
+     */
+    public BlockConstructionTapeCorner()
+    {
+        super(Material.VINE);
+        initBlock();
+    }
+
+
+    /**
+     * initialize the block
+     * sets the creative tab, as well as the resistance and the hardness.
+     */
+    private void initBlock()
+    {
+        setRegistryName(BLOCK_NAME);
+        setUnlocalizedName(String.format("%s.%s", Constants.MOD_ID.toLowerCase(), BLOCK_NAME));
+        setCreativeTab(ModCreativeTabs.MINECOLONIES);
+        GameRegistry.register(this);
+        GameRegistry.register((new ItemBlock(this)).setRegistryName(this.getRegistryName()));
+        this.setDefaultState(this.blockState.getBaseState().withProperty(FACING, EnumFacing.NORTH));
+        setHardness(BLOCK_HARDNESS);
+        setResistance(RESISTANCE);
+        setLightOpacity(LIGHT_OPACITY);
+        blockMaterial.setReplaceable();
+    }
+
+    @Nullable
+    @Override
+    public Item getItemDropped(final IBlockState state, final Random rand, final int fortune)
+    {
+        return null;
+    }
+    
+    /**
+     * @deprecated (Remove this as soon as minecraft offers anything better).
+     */
+
+    @Override
+    @Deprecated
+    public AxisAlignedBB getCollisionBoundingBox(IBlockState blockState, World worldIn, BlockPos pos)
+    {
+        return null;
+    }
+    
+    @Override
+    public int getMetaFromState(@NotNull final IBlockState state)
+    {
+        return state.getValue(FACING).getIndex();
+    }
+
+    /**
+     * Convert the given metadata into a BlockState for this Block.
+     *
+     * @deprecated (Remove this as soon as minecraft offers anything better).
+     */
+    @NotNull
+    @Override
+    @Deprecated
+    public IBlockState getStateFromMeta(final int meta)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1133)